src/tests/components/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { ErrorBoundary } from '../../components/UI'
+
+const ErrorText = 'Error message'
```

Comment:
The capitalized names should be used with components, containers, services or models names.

For variables, use: `errorText` and `errorChild`.

The snake case, use with *configurable* variables, ie. `API_KEY=xxxx`

Reply:
I will change it after merge because there is more file which needs to be changed